docs/niswitch/functions.rst

```diff
@@ -87,7 +87,7 @@ niswitch.Session methods
     semi-colons. For example, the following scan list will scan the first
     three channels of a multiplexer: com0->ch0; com0->ch1; com0->ch2; Refer
     to Scan Lists for more information on scan list syntax To see the status
-    of the scan, call either :py:func:`niswitch.is_scanning` or
+    of the scan, call either :py:func:`niswitch.IsScanning` or
```

Comment:
I'm confused as to why this shows up on your diff.

Reply:
I'm guessing that the generated code was not submitted from when we removed is_scanning from master. I made mention of it in the pull request above.

Reply:
Yeah that was it. I've updated generated files in master. If you merge again this will go away.